app/common/state/tabContentState/privateState.js

```diff
@@ -0,0 +1,36 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// State helpers
+const tabUIState = require('../tabUIState')
+const frameStateUtil = require('../../../../js/state/frameStateUtil')
+
+module.exports.isPrivateTab = (state, frameKey) => {
```

Comment:
Would it be better to add these methods to `app/common/tabState`?

These methods are getting information about a private tab... but that data would be in any tab.  Also, private tabs never have their state persisted... so having the file looks a little misleading

Reply:
makes sense will move them